front/app/modules/commercial/content_builder/admin/components/ContentBuilderTopBar/index.tsx

```diff
@@ -100,6 +101,18 @@ const ContentBuilderPage = ({ params: { projectId } }) => {
             </>
           )}
         </Box>
+        {!isNilOrError(project) && (
+          <Link to={`/projects/${project?.attributes.slug}`} target="_blank">
```

Comment:
Optional chaining (`?.`) is not necessary here

Reply:
Ah, true! I'll remove it. Thanks!